File: crates/siloscan-core/tests/profiles-corpus/tree/typescript/nesting-depth-over.ts
export function walkOver(values: number[]): number {
  let total = 0;
  for (const value of values) {
    for (const value of values) {
      for (const value of values) {
        for (const value of values) {
          for (const value of values) {
            for (const value of values) {
              total += value;
            }
          }
        }
      }
    }
  }
  return total;
}

File: crates/siloscan-core/tests/profiles-corpus/tree/go/nesting-depth-over.go
package p

func WalkOver(values []int) int {
	total := 0
	for _, value := range values {
		for _, value := range values {
			for _, value := range values {
				for _, value := range values {
					for _, value := range values {
						for _, value := range values {
							total += value
						}
					}
				}
			}
		}
	}
	return total
}
